CLAUDE.md

```
@@ -0,0 +1,236 @@
+- Implement benchmark tests with `b.Loop()` (Go 1.24)
+
+#### Linter Configuration
+Essential linters to run:
+- `durationcheck`: Detect incorrect time.Duration operations
+- `errcheck`: Ensure error returns are checked
+- `errorlint`: Enforce errors.Is/errors.As usage
+- `gocognit`: Keep cognitive complexity manageable
+- `gocritic`: Detect code improvement opportunities
+- `gosimple`: Simplify code
+- `ineffassign`: Detect ineffectual assignments
+- `staticcheck`: Wide range of code improvements
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)